tests/loops/test_evaluation_loop.py

```diff
@@ -42,25 +42,6 @@ def test_on_evaluation_epoch_end(eval_epoch_end_mock, tmpdir):
     assert eval_epoch_end_mock.call_count == 4
 
 
-@mock.patch(
-    "pytorch_lightning.trainer.connectors.logger_connector.logger_connector.LoggerConnector.log_eval_end_metrics"
-)
-def test_log_epoch_metrics_before_on_evaluation_end(update_eval_epoch_metrics_mock, tmpdir):
```

Comment:
This test is not useful anymore because logging results can be computed dynamically. This wasn't the case when it was added: https://github.com/PyTorchLightning/pytorch-lightning/issues/7166